src/storage/heap_file.c

```diff
@@ -22070,6 +22038,9 @@ heap_update_bigone (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context, b
   /* Proceed with the update. the new record is prepared and for mvcc it should have the prev version lsa set */
   if (heap_is_big_length (context->recdes_p->length))
     {
+      log_append_empty_record (thread_p, LOG_START_ATOMIC_REPL, NULL);
```

Comment:
Is it for RVHF_SET_PREV_VERSION_LSA?
I don't remember exactly, but the commit-msg is saying that we don't need it in this case.
> REC_BIGONE -> * : The BIGONE redo record already has the previous version lsa, so we don't need to care about this case.

Please tell me why we must pack it with the atomic operation.

Reply:
It seems to me that the record already contains the previous version lsa in 22034L.
`or_mvcc_set_log_lsa_to_record (context->recdes_p, logtb_find_current_tran_lsa (thread_p));`

Reply:
Oh, it's for the case in which updating a heap record must be atomic! Sorry, my bad.